mysql-test/suite/rpl/t/bug80821.test

```diff
@@ -0,0 +1,45 @@
+--source include/not_ndb_default.inc
+--source include/not_group_replication_plugin.inc
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+--source include/rpl_connection_master.inc
+
+CREATE TABLE t0 (
+f0 INT PRIMARY KEY
+) engine=innodb;
+
+CREATE TABLE t1 (
+f1 INT PRIMARY KEY,
+f0 INTEGER,
+FOREIGN KEY (f0)
+REFERENCES t0(f0)
+ON DELETE CASCADE
+) engine=innodb;
+
+INSERT INTO t0 VALUES (0);
+INSERT INTO t0 VALUES (1);
+INSERT INTO t1 VALUES (0, 0);
+INSERT INTO t1 VALUES (1, 0);
+
+# Should delete both t1 values, and the 0 value from t0
+DELETE t0a.*, t1.* FROM t0 AS t0a, t1 WHERE t0a.f0 = 0 AND t1.f1 = 0;
+
+--source include/sync_slave_sql_with_master.inc
+--source include/rpl_connection_master.inc
+
+SELECT COUNT(*) = 1 FROM t0;
```

Comment:
"sync slave to master, switch to master, check master, switch to slave, check slave"
could be simplified to
"check master, sync slave to master, check slave"